externs/shaka/player.js

```diff
@@ -1127,9 +1128,10 @@ shaka.extern.ManifestConfiguration;
  *   time for live streams. This can put us further from the live edge. Defaults
  *   to <code>false</code>.
  * @property {number} gapDetectionThreshold
- *   TThe maximum distance (in seconds) before a gap when we'll automatically
- *   jump. This value  defaults to <code>0.1</code>, except in Edge Legacy,
- *   Tizen, Chromecast that value defaults value is <code>0.5</code>
+ *   The maximum distance (in seconds) before a gap when we'll automatically
+ *   jump. This value  defaults to <code>0.5</code>.
+ * @property {numebr} gapJumpTimerTime
+ *   The polling time in seconds to check for gaps in the media.
```

Comment:
Could you mention the default of 0.25 seconds? It'd be helpful, because it implicitly tells the person reading the docs what the unit of the property is (seconds, not milliseconds), in addition to giving them a sense of what the default behavior is obviously.

Reply:
Done!